### ISSUE TITLE
Revision of Semester 1 code

### ISSUE THREAD

**Issue by taylorajez** [GamesTwoSpring2015/Project2.5D#9]
@sofoj1

**Comment by sofoj1** [comment]
I really think we should try to re implement the oxygen tanks, with varied level pickups (1/3, 2/3, 3/3). This could probably fix the balancing problem.

**Comment by taylorajez** [comment]
I agree, what you were talking about last class would seem to work. Maybe this is a task for the coming meeting.
